core/commands/cat.go

```diff
@@ -42,10 +41,13 @@ var CatCmd = &cmds.Command{
 			return
 		}
 
-		if err := corerepo.ConditionalGC(req.Context(), node, length); err != nil {
-			res.SetError(err, cmds.ErrNormal)
-			return
-		}
+		/*
+			if err := corerepo.ConditionalGC(req.Context(), node, length); err != nil {
+				res.SetError(err, cmds.ErrNormal)
+				return
+			}
+		*/
+
```

Comment:
This could have just been removed. We have git history and the changelog, and this code isn't really complicated enough to keep it for future reference :)